dashboard/src/components/PackageDetailsCard.vue

```diff
@@ -15,6 +15,11 @@ const download = () => {
 const stored = computed(() => {
   return packageStore.current?.aipId?.length;
 });
+
+watch(
+  () => packageStore.ui.download.watch(),
+  () => download()
+);
```

Comment:
Does the source need to be a function? Could it be `packageStore.ui.download.watch()` directly?